ts/screens/authentication/NewOptInScreen.tsx

```diff
@@ -95,15 +93,7 @@ const NewOptInScreen = () => {
       }}
     >
       <ContentWrapper>
-        {/*
-          if the device height is > 820 then the pictogram will be visible,
-          otherwise it will not be visible
-          */}
-        {Dimensions.get("screen").height > MIN_HEIGHT_TO_SHOW_FULL_RENDER && (
-          <View style={IOStyles.selfCenter} testID="pictogram-test">
-            <Pictogram name="passcode" size={120} />
-          </View>
-        )}
+        <PictogramOnSmallDevice name="passcode" />
         <VSpacer size={24} />
```

Comment:
On small devices, the pictogram will be hidden, but the `VSpacer` won't. We should probably replace it with the `VStack` component to avoid the extra space rendering when if the `Pictogram` is hidden.